### DISCUSSION TITLE
Fresh install on laravel 11 failed

### DISCUSSION THREAD

**Discussion by yudistirasd** [ivanwilliammd/satusehat-integration#134] [Q&A (Tanya Jawab)] [answered]
apa memang belum support laravel 11 pak @ivanwilliammd ?
![image](https://github.com/ivanwilliammd/satusehat-integration/assets/25135334/73026a93-effb-4b7f-9971-cb774fe8a751)

**Comment by ivanwilliammd** [comment]
harusnya bisa @yudistirasd , tapi memang belum di specify di composer.json

sudah saya fix di v2.8.0 ya dengan composer.json

**Comment by yudistirasd** [comment]
sudah saya coba masih sama pak

>  Problem 1
    - ivanwilliammd/satusehat-integration[0.14, ..., 0.15, 1.0, ..., 1.1] require illuminate/config ^10.25 -> found illuminate/config[v10.25.0, ..., v10.48.14] but these were not loaded, likely because it conflicts with another require.
    - ivanwilliammd/satusehat-integration[1.2.0, ..., 1.2.2, 2.0.0, ..., 2.8.0] require illuminate/config ^8.0|^9.0|^10.00 -> found illuminate/config[v8.0.0, ..., v8.83.27, v9.0.0, ..., v9.52.16, v10.0.0, ..., v10.48.14] but these were not loaded, likely because it conflicts with another require.
    - Root composer.json requires ivanwilliammd/satusehat-integration * -> satisfiable by ivanwilliammd/satusehat-integration[0.14, 0.15, 1.0, ..., 1.2.2, 2.0.0, ..., 2.8.0].
    
php artisan about :

>   Environment 
>   Application Name Laravel  
>   Laravel Version :  11.13.0  
>   PHP Version : 8.2.13  
>   Composer Version : 2.7.1

**Comment by ivanwilliammd** [comment]
ada install library lainnya kah?
ini sprtinya bukan masalah di library ini, tapi ke tabrakan dengan requirement library lain

**Comment by ivanwilliammd** [comment, marked as answer]
Barusan sdh saya recheck, ternyata dependencies illuminate Laravel juga naik, dan sdh saya fix di composer.json

silakan dicoba, di test dan dilaporkan kembali hasil testingnya v2.8.3 di framework laravel 11

**Comment by yudistirasd** [comment]
solve pak, terima kasih
